yappi/_yappi.c

```diff
@@ -224,7 +224,7 @@ static void _DebugPrintObjects(unsigned int arg_count, ...)
 
 int 
 IS_SUSPENDED(PyFrameObject *frame) {
-#if PY_MAJOR_VERSION >= 3 && PY_MINOR_VERSION == 11
+#if PY_MAJOR_VERSION >= 3 && PY_MINOR_VERSION >= 11
```

Comment:
This is unlikely to be enough, according to a comment below there are APIs in Python 3.12 to do that now?